src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -505,7 +505,7 @@ private function readProperty($zval, $property)
             if (\PHP_VERSION_ID >= 70400 && preg_match('/^Typed property ([\w\\\]+)::\$(\w+) must not be accessed before initialization$/', $e->getMessage(), $matches)) {
                 $r = new \ReflectionProperty($matches[1], $matches[2]);
 
-                throw new AccessException(sprintf('The property "%s::$%s" is not readable because it is typed "%3$s". You should either initialize it or make it nullable using "?%3$s" instead.', $r->getDeclaringClass()->getName(), $r->getName(), $r->getType()->getName()), 0, $e);
+                throw new AccessException(sprintf('The property "%s::$%s" is not readable because it is typed "%3$s". You should either initialize it or make it nullable using "?%3$s $var = null" instead.', $r->getDeclaringClass()->getName(), $r->getName(), $r->getType()->getName()), 0, $e);
```

Comment:
`You should either initialize it or make it default to null using "?%3$s $%2$s = null instead.`

Reply:
Yes it's better ;) PR feedback ok for me, thanks.